Assets/Scripts/Board/BoardController.cs

```diff
@@ -17,7 +17,8 @@ public int CreatedLines
         #endregion
 
         // Scripts References
-        BoardGenerate boardGenerate;
+        [HideInInspector]
+        public BoardGenerate boardGenerate;
 
         [Header("Board Settings")]
         public BoardStatusTypes boardStatusTypes;
```

Comment:
Devemos evitar o uso de atributos públicos em classes, para aumentar a segurança e a confiabilidade das classes. Isso também contribui para restringir quem acessa ou modifica cada atributo, facilitando o debug. Existem alguns motivos para usar atributos públicos, que podem ser cobertos por alternativas:

- Mostrar e modificar o atributo no Inspector;
  - O atributo pode ser privado e marcado com [SerializeField], presente no UnityEngine.
- Ser acessado e alterado por classes de fora;
  - O atributo pode ser privado e [usado como _backing field_ em uma propriedade](https://docs.microsoft.com/pt-br/dotnet/csharp/programming-guide/classes-and-structs/properties), ou um método, permitindo a implementação de regras especiais de acesso e/ou validações de valores.